src/shogun/features/SparseFeatures.h

```diff
@@ -48,6 +48,8 @@ template <class T> class CCache;
  */
 template <class ST> class CSparseFeatures : public CDotFeatures
 {
+	using CDotFeatures::dense_dot;
```

Comment:
SparseFeatures has its own `dense_dot` function (with different signature), which masks over `CDotFeatures::dense_dot`, so this is to enable both of them.